src/js/utils/useNavigation.js

```diff
@@ -46,7 +46,7 @@ const useNavigation = () => {
   const { pathname } = location;
   const { activeQuickStartID } = useContext(QuickStartContext);
   const currentNamespace = pathname.split('/')[1];
-  const schema = useSelector(({ chrome: { navigation } }) => navigation[currentNamespace]);
+  const [schema] = useSelector(({ chrome: { navigation, activeApp } }) => [navigation[currentNamespace], activeApp]);
```

Comment:
I had to check for active app to trigger re-render. Otherwise the nav wouldn't change :disappointed: